usaspending_api/references/v2/views/filter_tree/psc_filter_tree.py

```diff
@@ -15,18 +15,215 @@
     "Product": {"pattern": r"^\d\d$", "expanded_terms": [[digit] for digit in digits]},
 }
 
+PSC_GROUPS_COUNT = {
+    # A
+    "Research and Development": {"pattern": r"^A...$", "expanded_terms": ["A"]},
+    # B - Z
+    "Service": {
+        "pattern": r"^[B-Z][A-Z0-9][A-Z0-9][A-Z0-9]$",
+        "expanded_terms": [letter for letter in ascii_uppercase if letter != "A"],
+    },
+    # 0 - 9
+    "Product": {"pattern": r"^\d\d\d\d$", "expanded_terms": [digit for digit in digits]},
+}
```

Comment:
I'm seeing a lot of hard-coded "A"s down below. That can probably be incorporated into these constants (or PSC_GROUPS. Why are they both needed?)

Reply:
that's fair